pennylane_lightning_kokkos/lightning_kokkos.py

```diff
@@ -167,7 +339,7 @@ def apply_cq(self, operations, **kwargs):
                 param = o.parameters
                 method(wires, inv, param)
 
-    def apply(self, operations, **kwargs):
+    def apply(self, operations, rotations=None, **kwargs):
 
         # State preparation is currently done in Python
         if operations:  # make sure operations[0] exists
```

Comment:
We also need to check that the fall-back device exists and follows the same structure as `LightningGPU` (check [here](https://github.com/PennyLaneAI/pennylane-lightning-gpu/blob/main/pennylane_lightning_gpu/lightning_gpu.py#L801) to see more). In this way, if the kokkos device fails to load any module, the user gets a `LightningQubit` fallback.

Reply:
done!